### DISCUSSION TITLE
Reorder using drag and drop

### DISCUSSION THREAD

**Discussion by sedubois** [avo-hq/avo#2540] [Feature requests]
The [reordering](https://docs.avohq.io/3.0/records-reordering.html) feature seems to only make use of "move to top", "move to bottom", "move higher" and "move lower" buttons, but there does not seem to be an option to just drag a record to somewhere else in the list, nor an option to choose from a dropdown where the record should repositioned to (e.g. "move this record after article X" - this second option might be helpful with long paginated lists).

Also the documentation page and accompanying video seem to have been made with Avo 1, which made me wonder if the situation is still the same in the current Avo. Perhaps it would be enough to just refresh the screenshot to show the current Avo 3 UI with reordering.

**Comment by adrianthedev** [comment]
We just merged https://github.com/avo-hq/avo/pull/2574.
It should be out in the next release.

Wow. Such an old video!
I gotta redo it!

We would like to merge a PR where you would get an input of where should that record be inserted.
